gpkit/constraints/relax.py

```diff
@@ -30,13 +31,25 @@ def __init__(self, constraints):
         if not isinstance(constraints, ConstraintSet):
             constraints = ConstraintSet(constraints)
         substitutions = dict(constraints.substitutions)
-        posynomials = constraints.as_posyslt1()
+        relcons = []
         with NamedVariables("Relax"):
             self.relaxvar = Variable("C")
-        ConstraintSet.__init__(self,
-                               [[p <= self.relaxvar for p in posynomials],
-                                self.relaxvar >= 1], substitutions)
-
+        with SignomialsEnabled():
+            for constr in constraints.flat(constraintsets=False):
```

Comment:
consider using the more readable loop variable `constraint` instead of `constr`

Reply:
It was for pylint... I didn't want all of the lines to be broken.